snippets/AlkFejlHfTanulsagok/41/index.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: default
+codename: 
+title: Team-Dwayne csapat tanulságai
+tags: snippets
```

Comment:
```suggestion
tags: alkfejl afhf skipfromindex
```

Reply:
Megváltoztattam a mappa nevét, valamint a tags-et is.